AsyncDisplayKit/ASVisibilityProtocols.m

```diff
@@ -0,0 +1,23 @@
+//
+//  ASVisibilityProtocols.m
+//  Pods
+//
+//  Created by Garrett Moon on 4/28/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+#import "ASVisibilityProtocols.h"
+
+ASLayoutRangeMode ASLayoutRangeModeForVisibilityDepth(NSUInteger visibilityDepth)
```

Comment:
Why the separation between an integer value and the enum? Why not use the enum everywhere?

Really wish we could use Swift in these instances. A lot of what you're doing can be encapsulated into a single enum structure.